sycl/include/CL/sycl/interop_handle.hpp

```diff
@@ -167,9 +176,7 @@ class interop_handle {
       https://en.cppreference.com/w/cpp/language/reinterpret_cast
       https://en.cppreference.com/w/cpp/language/explicit_cast
       */
-    return (typename interop<BackendName,
-                             accessor<DataT, Dims, Mode, Target, IsPlh>>::type)(
-        getNativeMem(Req));
+    return (backend_return_t<Backend, buffer<DataT, Dims>>)(getNativeMem(Req));
```

Comment:
Should a `typename` be present here?

Reply:
If I understand it right, then 'typename' is not needed here because 'backend_return_t' is a 'using' here it is not a dependent typename, it is a non-dependent typename.